sdk/monitor/monitor-opentelemetry-exporter/src/types.ts

```diff
@@ -12,6 +12,7 @@ export type Tags = { [key in ContextTagKeys]: string };
  * Azure Monitor envelope property type.
  * @internal
  */
+// eslint-disable-next-line @typescript-eslint/ban-types
 export type PropertyType = string | number | boolean | object | Array<PropertyType>;
```

Comment:
I think this is still in beta, is there any chance to replace `object` with something better? /cc @xirzec

Reply:
`unknown` is better than object I think?

Reply:
Yeah, going with `unknown` makes sense if this is still in beta then.